tutorials/scripting/gdextension/files/cpp_example/SConstruct

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
```

Comment:
I am not sure the docs are the best place to host and maintain this file.

If this doesn't already exist i.e. in the GDExtension repository/example, we should create one, or maybe add it to the demo repository.

Reply:
I suppose you are right. It is mainly based on the test SConstruct file from here: https://github.com/godotengine/godot-cpp/blob/master/test/SConstruct

Reply:
I agree with @mhilbrunner, we should host the files used here in a repo of its own or find a way to incorporate the test project from godot-cpp in here.